native_client/deepspeech.h

```diff
@@ -163,9 +163,12 @@ int DS_EnableExternalScorer(ModelState* aCtx,
 /**
  * @brief Add a hot-word and its boost.
  *
+ * Excessive positive boost might lead to splitting up of letters of the word following the hot-word.
+ * Words that don't occur in the scorer (e.g. proper nouns) or strings that contain spaces won't be taken into account.
```

Comment:
This sentence should stay here (and it applies to all bindings where I said "Same")